README.md

```diff
@@ -1,2 +1,12 @@
-databasekurs
+Databasekurs
 ============
+
+Beskrivelse
+------------
+Database er ikke bare database lenger, de kommer i alle former og fasonger. Vi gir dere et innføring i og opplæring i to vidt forskjellige databaser som gir dere helt nye muligheter utover hva tradisjonelle relasjonelle databaser kan tilby.
+
+Første kveld gir vi dere en intro til MongoDB. Det første man kan bite seg merke i med MongoDB er enkelheten. Denne databasen skinner virkelig når man trenger å sette sammen noe i en fart!
+
+Kveld to er det Neo4j som er i søkelyset. Dette er en grafdatabase som lar deg gjøre veldig kraftige spørringer ut i fra et punkt i ett datasett.
```

Comment:
Lot den være litt ullen med vilje. Håpte @porqpine eller @olemartin  hadde en bedre salgspitch for Neo4j!